master/buildbot/steps/artifact.py

```diff
@@ -298,6 +362,13 @@ def rsyncWithRetry(step, origin, destination, port=None):
 
     return retryCommandLinuxOS(rsync_command)
 
+def mkDir(step):
+    if _isWindowsSlave(step):
+        return r'C:\cygwin64\bin\mkdir.exe'
```

Comment:
@MindWrapper I guess we need to check with @opdude / @jwebbintx if there are ok with this location on the agents (I'm not sure if this is consistent among windows setup) or what works best for our setup (make it configurable?)